extension/lefthook.yml

```diff
@@ -0,0 +1,6 @@
+pre-commit:
```

Comment:
We have own pre-commit config
https://github.com/hplush/slowreader/tree/main/.husky
https://github.com/hplush/slowreader/blob/main/nano-staged.json

Can you remove this?